### ISSUE TITLE
go-outliner install error, "go not found", "exe" must be in a bin directory, windows scoop install does not work

### ISSUE THREAD

**Issue by AddictArts** [766b/vscode-go-outliner#10]
Windows 10

The initial go get ... for outline was successful and go-outline.exe is in my home folder go/bin.

After clicking install the notification says cannot find go binary

**Comment by 766b** [comment]
Do you have path env defined for go/bin?

**Comment by AddictArts** [comment]
@766b Yes I do. I found a way to fix the issue for myself, but I do believe vscode-go-outliner should consider using GOROOT to find the go(.exe). Or vscode-go-outliner should not assume it needs to append bin to find go(exe).

Lines 320 and 321 in app.ts
```js
let appendBin = dirs[dirs.length - 1].toLowerCase() !== "bin";
let filePath = path.join(paths[i], appendBin ? 'bin' : '', toolFileName);
```

I used scoop for windows (think brew on Mac) to install go. Scoop uses a "shim" to point to the go(.exe).  So, from a command prompt I could see go.exe in my path; However, it is not in a ends with bin directory thus making the extension fail to find it.

```
...\scoop\shims\go.exe
```
